packages/editor-ui/src/components/Modal.vue

```diff
@@ -141,22 +145,18 @@ export default Vue.extend({
 				this.$emit('enter');
 			}
 		},
-		closeDialog(callback?: () => void) {
+		closeAllDialogs() {
+			this.$store.commit('ui/closeAllModals');
+		},
+		async closeDialog() {
```

Comment:
avoid callbacks. refactor to be more specific when closing modals, rather than simply closing top one.